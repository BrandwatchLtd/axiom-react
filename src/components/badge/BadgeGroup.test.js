import React from 'react';
import renderer from 'react-test-renderer';
import { BadgeGroup, Badge } from 'bw-axiom';

function getComponent(props = {}) {
  return renderer.create(
    <BadgeGroup { ...props }>
      <Badge>Lorem ipsum</Badge>
    </BadgeGroup>
  );
}

describe('BadgeGroup', () => {
  it('renders with defaultProps', () => {
    const component = getComponent();
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
