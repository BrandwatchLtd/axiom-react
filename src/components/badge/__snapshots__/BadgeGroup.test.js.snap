// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BadgeGroup renders with defaultProps 1`] = `
<div
  className="ax-badge-group__container ax-space--small"
>
  <div
    className="ax-badge-group"
  >
    <span
      className="ax-badge ax-badge--faded"
    >
      Lorem ipsum
    </span>
  </div>
</div>
`;
